openapi-v1.yaml

```diff
@@ -2854,6 +2854,58 @@ definitions:
       pagination_metadata:
         $ref: "#/definitions/PaginationMetadata"
 
+  InvitationDetails:
+    description: Object including invitation details such as the array shared or organization
+    type: object
+    properties:
+      array:
+        description: the array shared
+        x-omitempty: true
+        properties:
+          id:
+            type: string
+            description: array id
+          namespace:
+            type: string
+            description: the namespace that the array belongs to
+          name:
+            type: string
+            description: name of the array
+          file_type:
```

Comment:
`file_type` is internal. Maybe a better name like `asset_name` or `asset_type` or whatever

Reply:
file_type is a property of ArrayInfo definition